war/src/main/less/modules/manage-jenkins.less

```diff
@@ -1,152 +1,175 @@
-    margin-left: 15px;
+  margin-left: 15px;
 }
 
 .manage-messages dl:last-child {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
 
 .manage-messages dl dt::after {
-    content: ": ";
+  content: ": ";
+}
+
+.manage-messages {
+  .alert:last-of-type {
+    margin-bottom: 30px;
+  }
 }
 
 .manage-messages .alert a {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert a:hover {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert form {
-    position: relative;
-    float: right;
-    margin: -6px 0 0 0 !important;
+  position: relative;
+  float: right;
+  margin: -6px 0 0 0 !important;
 }
 
 .manage-messages .alert form span {
-    margin: 0 0 0 4px !important;
+  margin: 0 0 0 4px !important;
 }
 
-.manage-option {
-    border: 1px solid transparent;
-    margin-bottom: 4px;
+.jenkins-settings {
+  &__section-title {
+    margin: 0 0 calc(2rem) 0;
+    font-size: 0.8rem;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    opacity: 0.75;
+  }
+
+  &__section {
+    border-top: 2px solid var(--panel-border-color);
+    padding: calc(2rem) 0;
+    max-width: 1800px;
 
-    &:hover,
-    &:focus-within {
-      border: 1px solid var(--manage-option-border-color);
+    &:first-of-type {
+      border-top: none;
+      padding-top: 0;
     }
-}
 
-.manage-option dl {
-    padding: 0;
-    margin: 0 0 0 62px;
-    min-height: 48px;
-}
+    .manage-page__row {
+      display: grid;
+      grid-template-columns: 1fr;
+      grid-gap: 30px;
 
-.manage-option dl dt {
-    font-size: var(--font-size-base);
-}
+      @media screen and (min-width: 800px) {
+        grid-template-columns: 1fr 1fr;
+      }
 
-.manage-option dl dd {
-    margin-left: 0;
-    line-height: 20px;
-    color: var(--text-color);
-    font-size: var(--font-size-sm);
+      @media screen and (min-width: 1300px) {
+        grid-template-columns: 1fr 1fr 1fr;
+      }
+
+      @media screen and (min-width: 1800px) {
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+      }
+    }
+    
+    &:last-child {
+      padding-bottom: 0;
+    }
+  }
 }
 
 .manage-option a {
-    display: block;
-    padding: 8px;
+  position: relative;
+  display: flex;
+  z-index: 0;
+  text-decoration: none;
+
+  // IE11 doesn't support grid-gap, so apply margin, remove it for modern browsers
+  margin: 0 30px 30px 0;
+  @supports (grid-gap: 30px) {
     margin: 0;
-    text-decoration: none !important;
-    color: var(--text-color);
-    font-weight: normal;
-    height: 100%;
+  }
 
-    &:hover,
-    &:focus {
-      background-color: var(--manage-option-bg-color--hover);
-    }
+  &::before {
+    position: absolute;
+    content: "";
+    top: -10px;
+    left: -10px;
+    bottom: -10px;
+    right: -10px;
+    z-index: -1;
+    border-radius: 5px;
+  }
 
-    &:focus {
-      outline-color: var(--focus);
+  &:hover,
+  &:focus {
+    &::before {
+      background-color: var(--btn-link-bg--hover);
```

Comment:
Can't remember now, but doesn't this also have variables for button link color and border?